test/common/thread_local/thread_local_impl_test.cc

```diff
@@ -47,6 +47,9 @@ class ThreadLocalInstanceImplTest : public testing::Test {
   InstanceImpl tls_;
   Event::MockDispatcher main_dispatcher_;
   Event::MockDispatcher thread_dispatcher_;
+  std::condition_variable cv_;
```

Comment:
Can you spell out "condvar_" in the variable rather this abbreviation?

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

More importantly, this is only used in one test, so can you move these variables into the test?  You could put them in a local struct there if you like, so that it's easier to capture them all at once for the nested function.

Reply:
moved them to a local struct and renamed the variables. Can you PTAL at this specific test case and see if that is what you are looking for?